src/vm/frames.h

```diff
@@ -418,6 +418,10 @@ class Frame : public FrameBase
 
 public:
 
+#if defined(FEATURE_PAL) && !defined(DACCESS_COMPILE) 
```

Comment:
In my opinion, ifdefining members in .h file is bad practice. I suggest to declare member as it is, but idef it's body, not declaration